content/en/account_management/rbac/permissions.md

```diff
@@ -0,0 +1,372 @@
+| logs_write_pipelines         | Update a subset of the log pipelines       | true     |
+| logs_write_processors        | Update the log processors in an index      | true     |
+| logs_write_archives          | Update the external archives configuration | false    |
+| logs_public_config_api       | Access the Logs Public Config API (r/w)    | false    |
+| logs_generate_metrics        | Access the Generate Metrics feature        | false    |
+
+More details about these permissions below.
+
+#### Configuration
+
+##### logs_live_tail
+
+Grants a role the ability to use the live tail feature.
+
+{{< tabs >}}
+{{% tab "Datadog application" %}}
+
+Go to your [Datadog Roles page][1] and select the checkbox `read` as below for the wanted role:
+{{< img src="account_management/rbac/logs_livetail_access.png" alt="Create a custom Role"  style="width:90%;">}}
+
+
+[1]: https://app.datadoghq.com/access/roles
+{{% /tab %}}
+{{% tab "API" %}}
+
+This permission can be granted or revoked from a role via [the Roles API][1].
+
+
+[1]: /api/#roles
+{{% /tab %}}
+{{< /tabs >}}
+
+##### logs_generate_metrics
+
+Grants a role the ability to use the Generate Metrics feature. This permission is global and applies to the configuration of all the metrics generated from logs.
+
+{{< tabs >}}
+{{% tab "Datadog application" %}}
+
+Go to your [Datadog Roles page][1] and select the checkbox `other` as below for the wanted role:
+{{< img src="account_management/rbac/logs_generate_metrics_access.png" alt="Create a custom Role"  style="width:90%;">}}
+
+
+[1]: https://app.datadoghq.com/access/roles
+{{% /tab %}}
+{{% tab "API" %}}
+
+This permission can be granted or revoked from a role via [the Roles API][1].
+
+
+[1]: /api/#roles
+{{% /tab %}}
+{{< /tabs >}}
+
+##### logs_modify_indexes
+
+Grants a role the ability to create and modify log indexes. This includes:
+
+- Setting inclusion queries for which logs should be routed into an index.
+- Setting log retention for an index.
+- Limiting which roles have read access on an index (`logs_read_index_data`).
+- Which roles can modify exclusion filters for an index (`logs_write_exclusion_filters`).
+
+{{< tabs >}}
+{{% tab "Datadog application" %}}
+
+Go to your [Datadog Roles page][1] and select the checkbox `other` as below for the wanted role:
+{{< img src="account_management/rbac/logs_modify_indexes_access.png" alt="Create a custom Role"  style="width:90%;">}}
+
+
+[1]: https://app.datadoghq.com/access/roles
+{{% /tab %}}
+{{% tab "API" %}}
+
+This permission can be granted or revoked from a role via [the Roles API][1].
+
+
+[1]: /api/#roles
+{{% /tab %}}
+{{< /tabs >}}
+
+##### logs_write_exclusion_filters
+
+Grants a role the ability to create or modify exclusion filters within an index. This can be assigned either globally or restricted to a subset of indexes.
+
+{{< tabs >}}
+{{% tab "Datadog application" %}}
+
+**Global access**:
```

Comment:
The bolded **Global access** ends up looking bigger than the header it is under (`##### logs_write_exclusion_filter`). Is there a way to make it smaller? maybe it makes sense to try `###### Global Access`. 

not a big deal, just looks a bit funny.

Reply:
I'm actually fine to insist on the global vs local. As I believe this is very important.

What do you think?

Reply:
Updated headers on the whole page to ease navigation and prepare further improvements